plugins/certs/commands

```diff
@@ -55,8 +61,8 @@ certs_set() {
   fi
 
   mkdir -p "$DOKKU_ROOT/$APP/tls"
-  mv "$CRT_FILE" "$DOKKU_ROOT/$APP/tls/server.crt"
-  mv "$KEY_FILE" "$DOKKU_ROOT/$APP/tls/server.key"
+  cp "$CRT_FILE" "$DOKKU_ROOT/$APP/tls/server.crt"
+  cp "$KEY_FILE" "$DOKKU_ROOT/$APP/tls/server.key"
```

Comment:
@michaelshobbs no you tell me, why you delete my certs bro? That was my only copy!!! :wink: